src/components/pages/doc/sidebar/item/item.jsx

```diff
@@ -4,77 +4,60 @@ import React, { useState } from 'react';
 
 import Link from 'components/shared/link';
 import { DOCS_BASE_PATH } from 'constants/docs';
-import ChevronRight from 'icons/chevron-right.inline.svg';
+import ChevronRight from 'icons/chevron-right-sm.inline.svg';
 
-import SubItem from './sub-item';
+const isActiveItem = (items, currentSlug) =>
+  !!items?.find(
+    ({ slug, items }) => slug === currentSlug || (items && isActiveItem(items, currentSlug))
+  );
 
 const Item = ({ title, slug, isStandalone, items, isOpenByDefault, currentSlug }) => {
-  const [isOpen, setIsOpen] = useState(isOpenByDefault);
+  const [isOpen, setIsOpen] = useState(isOpenByDefault || slug === currentSlug);
 
-  const handleClick = () => setIsOpen((isOpen) => !isOpen);
+  if (!isOpen && isActiveItem(items, currentSlug)) {
```

Comment:
`useState(isOpenByDefault || slug === currentSlug);`
and
`if (!isOpen && isActiveItem(items, currentSlug)) {`
I think there is something redundant here